frontend/src/components/short-car-card/styles.module.scss

```diff
@@ -0,0 +1,34 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 255px;
+  padding: 30px;
+  background-color: var(--white);
+  border-radius: 5px;
+  cursor: pointer;
```

Comment:
Please use the same CSS classes naming as in the `top-car-card` component

Reply:
@oleksandr-danylchenko I can't find this component in the project. Maybe you mean new-car-card component?

Reply:
I found this component. It wasn't in my branch when I created it. I added new changes from the develop branch and renamed classes